sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/fullfidelity/ChangeFeedContextClientImpl.java

```diff
@@ -93,7 +96,12 @@ public Flux<FeedResponse<PartitionKeyRange>> readPartitionKeyRangeFeed(String pa
     public Flux<FeedResponse<JsonNode>> createDocumentChangeFeedQuery(
         CosmosAsyncContainer collectionLink,
         CosmosChangeFeedRequestOptions changeFeedRequestOptions) {
+        throw new UnsupportedOperationException("createDocumentChangeFeedQueryV1() should be called instead for Full Fidelity");
+    }
 
+    @Override
+    public Flux<FeedResponse<ChangeFeedProcessorItem>> createDocumentChangeFeedQueryV1(
```

Comment:
hmm, can we use <T> for the parameter type instead of creating a new file?

Reply:
Good point. !